databricks_cli/libraries/cli.py

```diff
@@ -0,0 +1,263 @@
+from databricks_cli.version import print_version_callback, version
+
+
+def _all_cluster_statuses():
+    click.echo(pretty_format(all_cluster_statuses()))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Get the status of all libraries.')
+@require_config
+@eat_exceptions # noqa
+def all_cluster_statuses_cli():
+    """
+    Get the status of all libraries on all clusters. A status will be available for all libraries
+    installed on this cluster via the API or the libraries UI as well as libraries set to be
+    installed on all clusters via the libraries UI. If a library has been set to be installed on
+    all clusters, is_library_for_all_clusters will be true,
+    even if the library was also installed on this specific cluster.
+    """
+    _all_cluster_statuses()
+
+
+def _cluster_status(cluster_id):
+    click.echo(pretty_format(cluster_status(cluster_id)))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Get the status of all libraries for a specified cluster.')
+@click.option('--cluster-id', required=True, type=ClusterIdClickType(),
+              help=ClusterIdClickType.help)
+@require_config
+@eat_exceptions # noqa
+def cluster_status_cli(cluster_id):
+    """
+    Get the status of all libraries on all clusters. A status will be available for all libraries
+    installed on this cluster via the API or the libraries UI as well as libraries set to be
+    installed on all clusters via the libraries UI. If a library has been set to be installed on
+    all clusters, is_library_for_all_clusters will be true,
+    even if the library was also installed on this specific cluster.
+    """
+    _cluster_status(cluster_id)
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Alias of `all-cluster-statuses` or `cluster-status`.')
+@click.option('--cluster-id', type=ClusterIdClickType(), default=None,
+              help=ClusterIdClickType.help)
+@require_config
+@eat_exceptions # noqa
+def list_cli(cluster_id):
+    """
+    If the option --cluster-id is provided, then all libraries on that cluster will be listed,
+    (cluster-status). If the option --cluster-id is omitted, then all libraries on all clusters
+    will be listed (all-cluster-statuses).
+    """
+    if cluster_id is not None:
+        _cluster_status(cluster_id)
+    else:
+        _all_cluster_statuses()
+
+
+INSTALL_OPTIONS = ['jar', 'egg', 'maven-coordinates', 'pypi-package', 'cran-package']
+JAR_HELP = 'JAR uploaded to dbfs or s3.'
+EGG_HELP = 'Egg uploaded to dbfs or s3.'
+MAVEN_COORDINATES_HELP = """
+Maven coordinates in the form of GroupId:ArtifactId:Version (i.e. org.jsoup:jsoup:1.7.2).
+"""
+MAVEN_REPO_HELP = """
+Maven repo to install the Maven package from. If omitted, both Maven 
+Repository and Spark Packages are searched.
+"""
+MAVEN_EXCLUSION_HELP = """
+List of dependences to exclude. For example: ["slf4j:slf4j", "*:hadoop-client"].
```

Comment:
The example should be `--maven-exclusion "slf4j:slf4j" --maven-exclusion "*:hadoop-client"`